src/report.sh

```diff
@@ -173,6 +173,7 @@ function grouping_day_data()
   local timebox_sec
   local total_time_box_sec=0
   local total_repetition=0
+  local -A used
```

Comment:
Can you find a better name here? Fwiu, this variable works like a check if something is already in use or not, right?

Also, in the commit message, add a sample of how the output looks before and after your commit.

Reply:
The variable is used to check which tags have been accounted for in that day, to avoid printing the same date twice on each tag entry. I was heavily inspired by the *kw_include.sh* file.
What do you think of `date_printed`?